src/BlazorUI/Bit.BlazorUI/Components/Inputs/SearchBox/BitSearchBox.scss

```
@@ -524,6 +602,63 @@
+    overflow-wrap: break-word;
+}
+
+.bit-srb-spn {
+    flex-shrink: 0;
+    border-radius: 50%;
+    width: spacing(1.75);
+    height: spacing(1.75);
+    box-sizing: border-box;
+    border: spacing(0.25) $shp-border-style $clr-brd-pri;
+    border-block-start-color: $clr-pri;
+    // The spinner tokens rather than a literal duration: a looping animation must slow down
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)